applications/order/groovyScripts/shoppinglist/ShoppingListServices.groovy

```diff
@@ -131,39 +123,30 @@ def updateShoppingListItem() {
 
 /**
  * Remove a ShoppingListItem
- * @return
  */
-def removeShoppingListItem() {
-    GenericValue shoppingList = from("ShoppingList").where(parameters).queryOne()
-    GenericValue shoppingListItem = from("ShoppingListItem").where(parameters).queryOne()
+Map removeShoppingListItem() {
+    GenericValue shoppingList = from('ShoppingList').where(parameters).queryOne()
+    GenericValue shoppingListItem = from('ShoppingListItem').where(parameters).queryOne()
     shoppingListItem.remove()
 
     updateLastAdminModified(shoppingList, userLogin)
     return success()
 }
 
```

Comment:
Why moving this method?

Reply:
What method was removed ? I only see return type and GString to string modifications ?

Reply:
Not removed, moved

Reply:
Wait, I think I mixed with another file, checking....

Reply:
I speak about updateLastAdminModified()

Reply:
oh, indeed i missread, the if was inverted to respect InvertedIfElse rule : An inverted if-else statement is one in which there is a single if statement with a single else branch and the boolean test of the if is negated. For instance if (!x) false else true. It is usually clearer to write this as if (x) true else false.

The method was not moved since it is still in the  `if (!shoppingListItems) {` block.